src/librustc_resolve/lib.rs

```diff
@@ -835,7 +834,7 @@ pub struct Resolver<'a> {
 
     /// Names of fields of an item `DefId` accessible with dot syntax.
     /// Used for hints during error reporting.
-    field_names: FxHashMap<DefId, Vec<Spanned<Name>>>,
+    field_names: FxHashMap<DefId, Vec<Ident>>,
```

Comment:
This is not an identifier, it's a `Name` combined with an unrelated `Span`.
If we want to avoid `Spanned`, then this can be e.g. `(Name, Span)`.
The change is unrelated to this PR though.

Reply:
The `Span` is very much related - it's the exact span where the `Symbol` occurred (if it's a named field), or the span of the type position (if it's an unnamed field).

However, I see your point about this lacking the additional meaning of `Ident` (the second case, strictly speaking, isn't an identifier). I'll change this back.